packages/shared/src/components/feeds/FeedContainer.tsx

```diff
@@ -280,7 +288,8 @@ export const FeedContainer = ({
             <div
               className={classNames(
                 'grid',
-                isSearch && !shouldUseListFeedLayout && 'mt-8',
+                !isLaptop && (isExplorePopular || isExploreLatest) && 'mt-4',
+                isSearch && !shouldUseListFeedLayout && !isExplore && 'mt-8',
```

Comment:
Here as well wondering if we can uniform it somehow, not sure what the issue is design wise but we can suggest things to keep it neater.

Reply:
The thing here is that it should not be part of the `FeedLayout` itself, might sound counter-intuitive, but when the screen is empty, you lose the header tabs to navigate between the options. At first, I was rendering both of them at the `header` property, but then the explained case above surfaced.